pom.xml

```diff
@@ -1154,131 +1170,128 @@
+                                          zipOutputStream.closeEntry()
+                                      } catch (Exception error) {
+                                          log.error('failed processing file `' + it.key + '` at path `' + it.value + '`', error)
+                                          throw error
+                                      }
+                                  }
+
+                              }
+                          } catch (Exception error) {
+                              log.error('failed creating ZIP archive `' + zipFileName + '`', error)
+                              throw error
+                          }
+                      }
+
+                      // Find Git-tracked files
+                      SortedMap pathByFile = new TreeMap()
+                      Repository repo = new FileRepositoryBuilder().readEnvironment().findGitDir().build()
+                      DirCache repoCache = repo.readDirCache()
+                      String repoDirectoryParent = repo.getDirectory().getParent()
+                      for (int repoCacheEntryIndex = 0; repoCacheEntryIndex < repoCache.getEntryCount(); repoCacheEntryIndex++) {
+                          DirCacheEntry repoCacheEntry = repoCache.getEntry(repoCacheEntryIndex)
+                          String repoCacheEntryPath = repoCacheEntry.getPathString()
+                          pathByFile.put(repoCacheEntryPath, new File(repoDirectoryParent, repoCacheEntryPath).toPath())
+                      }
+
+                      // Create the source distribution using Git-tracked files
+                      zip(project.build.directory + '/src.zip', pathByFile)
+                      log.info('Generated the source distribution (`src.zip`) containing ' + pathByFile.size() + ' files.')
+
+                      // Short-circuit if there is no binary distribution expected
+                      if (attachmentCount == 0) {
+                          return
+                      }
+
+                      // Find auxiliary files that will go into the binary distribution
+                      pathByFile = new TreeMap()
+                      String releaseNotes = project.build.directory +
+                          '/generated-site/antora/modules/ROOT/pages/_release-notes/' +
+                          project.version + '.adoc'
+                      pathByFile.put("RELEASE-NOTES.adoc", Paths.get(releaseNotes))
+                      pathByFile.put("README.adoc", Paths.get("README.adoc"))
+                      pathByFile.put("NOTICE.txt", Paths.get("NOTICE.txt"))
+                      pathByFile.put("LICENSE.txt", Paths.get("LICENSE.txt"))
+
+                      // Find attachments that will go into the binary distribution
+                      Path projectDir = Paths.get(getProperty('maven.multiModuleProjectDirectory'))
+                      log.info("Locating attachments matching the provided pattern: ${d}{attachmentFilepathPattern}")
```

Comment:
Can we stick to one quoting style throughout all Groovy scripts, please? (I'd prefer double-quotes, to match the Java style followed in the rest of the code base.)

Reply:
Fixed in https://github.com/apache/logging-parent/pull/241/commits/40eb92c6c85f5b419280510ea27bb4c34cbc376c.